bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tmva/__init__.py

```diff
@@ -18,9 +18,16 @@
 from ._factory import Factory
 from ._dataloader import DataLoader
 from ._crossvalidation import CrossValidation
-from ._rbdt import Compute, pythonize_rbdt
-from ._rtensor import get_array_interface, add_array_interface_property, RTensorGetitem, pythonize_rtensor
-from ._tree_inference import SaveXGBoost, pythonize_tree_inference
+
+# We got to load the pythonization functions from the submodules explicitly
+# because the _pythonization/__init__.py doesn't automatically load modules
+# from subdirectories. There is also no harm in importing _rtensor when
+# RDataFrame is not enabled in the build, as the actual pythonization will only
+# be called lazily if an RTensor is instantiated, which can't happen if you
+# don't have RDataFrame enabled.
```

Comment:
What I would say here is that you need these imports to trigger the registration of the pythonizors for rbdt, rtensor and tree_inference. Is this registration done at import ROOT time or only when accessing the TMVA namespace?

Reply:
Also this might be better in a separate commit.

Reply:
No, these changes are not decoupled. In the previous `file(COPY...` command, the whole python source directory was copied, and now we explicitly use the `py_sources` list to copy only the source files. That's why now we have to make sure to list all the sources, which mattered only for the optional python compilation before so it was unnoticed that files were missing.

Reply:
Okay, I will update the PR to not touch that file, and then it's up to @lmoneta anyway to fix the RTensor stuff in his PR.